dev/tasks/conda-recipes/azure.clean.yml

```diff
@@ -1,3 +1,5 @@
+{% import 'macros.jinja' as macros with context %}
```

Comment:
Is importing the macros necessary? I thought that as we are not using the macros anywhere on this file it shouldn't be necessary.
Not related to your specific PR but I've noticed the License header is missing on this file, could you also add it?

Reply:
1. You are absolutely right! Importing the macros are not required anymore. This was an oversight on my part - I left this in by accident from a previous version of the code which used a jinja macro instead of a method on the `Target` object to check whether the current branch is the default branch. Since leaving these import statements in didn't cause any obvious issues, I completely forgot to remove them. My apologies! Thank you for catching this!

2. Of course - I will add the License header.

Reply:
After searching around a bit more - it looks like many of the files in the `/dev/tasks/conda-recipes` directory are missing license headers. It might be worth following up with a future PR to add them.